CHANGELOG.md

```diff
@@ -2,7 +2,13 @@
 
 ## v2.10.0
 
-- Share of Checkout
+- Optimise your performance with insights on share of checkout
```

Comment:
il faut reprendre les termes fix: ou feat: etc

Reply:
C'est le changelog qu'on avait validé à la dernière release du Share of Checkout :)